docs/installation.rst

```diff
@@ -19,7 +19,9 @@ Notes
 +--------------------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
 |**Python**          |**3.8**|**3.7**|**3.6**|**3.5**|**3.4**|**3.3**|**3.2**|**2.7**|**2.6**|**2.5**|**2.4**|
 +--------------------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+
-|Pillow >= 7         |  Yes  |  Yes  |  Yes  |  Yes  |       |       |       |       |       |       |       |
+|Pillow >= 7.3       |  Yes  |  Yes  |  Yes  |       |       |       |       |       |       |       |       |
```

Comment:
```suggestion
|Pillow >= 8.0       |  Yes  |  Yes  |  Yes  |       |       |       |       |       |       |       |       |
```

Also need to add Python 3.9 column, as it should be released the week before.

Reply:
Thanks, done!